lib/usdUfe/ufe/UsdSceneItemOps.cpp

```diff
@@ -78,13 +74,17 @@ bool UsdSceneItemOps::deleteItem()
 #ifdef UFE_V4_FEATURES_AVAILABLE
 Ufe::SceneItemResultUndoableCommand::Ptr UsdSceneItemOps::duplicateItemCmdNoExecute()
 {
-    return UsdUndoDuplicateCommand::create(_item);
+    auto hier = Ufe::Hierarchy::hierarchy(_item);
+    auto parentItem = hier->parent();
+    return UsdUndoDuplicateCommand::create(_item, downcast(parentItem));
```

Comment:
The UsdUfe duplicate command takes an extra item which is the destination for the duplicate. Use the parent which is what the MayaUsd duplicate command does by default.